package.json

```diff
@@ -4,7 +4,8 @@
     "description": "A place to store global brand information for use across various digital properties (Sass colour variables, typography etc)",
     "main": "index.js",
     "scripts": {
```

Comment:
Do you think it's worth having usage notes for these in the readme? (this is code for "idk what these are actually for" 😛 )